docs/docsite/rst/playbooks_tags.rst

```diff
@@ -86,27 +98,43 @@ You can apply tags to more than tasks, but they ONLY affect the tasks themselves
       tasks:
         ...
 
-You may also apply tags to roles::
+You may also apply tags to ``roles:``:
 
     roles:
       - { role: webserver, port: 5000, tags: [ 'web', 'foo' ] }
 
-And import statements::
+And to ``import_role:`` and ``import_tasks:`` statements::
+
+    - import_role:
+        name: myrole
+      tags: [web,foo]
 
     - import_tasks: foo.yml
       tags: [web,foo]
 
+
 All of these apply the specified tags to EACH task inside the play, imported
 file, or role, so that these tasks can be selectively run when the playbook
 is invoked with the corresponding tags.
 
-The above information does not apply to `include_tasks` or other dynamic includes,
-as the attributes applied to an include, only affect the include itself.
+Tags are inherited *down* the dependency chain. In order for tags to be
+applied to a role and all its dependencies, the tag should be applied to the
```

Comment:
clarify you mean 'dependant roles'

Reply:
This section now says "In order for a tag to be inherited to a dependent role's tasks, ..."